src/nav/NavigationContainer.js

```diff
@@ -2,28 +2,39 @@ import React from 'react';
 import { connect } from 'react-redux';
 
 import boundActions from '../boundActions';
+import { checkCompatibility } from '../api';
 import { getInitialRoutes } from './routingSelectors';
 import Navigation from './Navigation';
 
 class NavigationContainer extends React.PureComponent {
 
+  state = {
```

Comment:
Could you please add a comment or two that `compatibilityCheckFail` has to do with forced upgrades? Someone new to the code might be confused about its purpose